_posts/2021-02-05-status-update.md

```diff
@@ -0,0 +1,100 @@
+- [`pdk#939`][pdk-pr-939]: "(MAINT) Disable parallel unit package testing specs on Windows", thanks to [scotje][scotje]
+- [`pdk#932`][pdk-pr-932]: "Add ability to generate functions", thanks to [logicminds][logicminds]
+- [`pdk-vanagon#274`][pdk-vanagon-pr-274]: "Revert "Only install system_tests gem on the main ruby"", thanks to [scotje][scotje]
+- [`ruby-pwsh#101`][ruby-pwsh-pr-101]: "(MAINT) Place nil check when assigning is_same", thanks to [bwilcox][bwilcox]
+- [`ruby-pwsh#96`][ruby-pwsh-pr-96]: "(MAINT) Ensure is_same check works for nil manifest values", thanks to [bwilcox][bwilcox]
+
+Please swing by our [office hours on Monday](https://puppet.com/community/office-hours/) on Slack if you want to directly contact us about anything module related.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-stdlib`][puppetlabs-stdlib] (`6.6.0`)
+- [`puppetlabs-docker`][puppetlabs-docker] (`3.13.1`)
+- [`puppetlabs-pwshlib`][puppetlabs-pwshlib] (`0.7.3`)
+
+## Cloud CI
+
+This week the development work was focused on the automated release preparation and multi-node testing.
+
+The tempate for the automated module release preparation in `pdk-templates` is currently [undergoing testing][pdk-templates-pr-394] and, hopefully, by the end of next week a pdksync across all our supported modules will ease releases in the future.
+
+For a quick preview on multi-node testing, [take a look at this NTP pull request][puppetlabs-ntp-pr-603] that deploys two VMs on GCP with different roles and runs an integration test suite to check communications.
+
+## DSC
+
+Over the last two weeks we've released several small iterative improvements to our [`Puppet.Dsc` PowerShell Module][dsc-pwsh] and the [`puppetlabs-pwshlib` Puppet module][puppetlabs-pwshlib] as we draw towards closing out our [first major milestone][dsc-milestone-1].
+As part of this effort, we published two new blog posts; one on [troubleshooting Puppetized DSC modules][dsc-troubleshooting-blog] and another by [Paula][Paula] on [migrating from `puppetlabs-dsc` to the Puppetized `dsc-*` modules][dsc-migration-blog].
+
+The troubleshooting document has already paid off, leading to the filing of excellent bug reports and even a bugfix PR independent of the IAC team by [Bill Wilcox][bwilcox].
+It also led to a pairing session with [Jordan Bush][jordanbush] where he was able to discover and [file a repro for a bug in `xDnsServerDiagnostics`][dsc-bug-repro] in the [xDnsServer PowerShell module][dsc-xdnsserver].
+
+We're excited to see the adoption of the Puppetized modules in the `dsc` namespace on the Forge grow, especially now that we've cut down on some of the guess work for migrating from the older `puppetlabs-dsc` module.
+The last remaining tasks in our first milestone are administrative, so next week will see us tackling [issues in the next milestone][dsc-milestone-2] which should set us up for a `1.0.0` release.
+
```

Comment:
```suggestion

## PDK

Check out the new [nightly builds](http://nightlies.puppet.com/) (use the `puppet6-nightly` repos [for now](https://tickets.puppetlabs.com/browse/PDK-1683)) of the PDK with ruby 2.7, facter 4 and puppet 7. Please note that this is also dropping puppet 4 and hence will become PDK 2.0 at some point in the near future.

## Ecosystem Kickoffs

Despite the travel restrictions, the wider ecosystem group at Puppet came together this week to share challenges, ideas, and plans. While we can't share many of the details (yet!!), it was a great opportunity to exchange with our friends from the Forge, Bolt, Integrations, and Solution teams, as well as update our understanding of how our work fits into the greater Puppet Product Strategy. Big shout-out to [Ben Ford](https://twitter.com/binford2k) for stepping up to the role of Product Manager and organising a week chock-full of Content!

## Litmus
Last week, [David Swan][DavidSwan] addressed a long-standing issue where litmus would get confused about exit codes when running puppet. This week saw a minor adjustment and another [litmus release](https://github.com/puppetlabs/puppet_litmus/releases/tag/v0.23.0) to make this also work for the `docker_exp` provisioner.

## Cisco Networking

[Ciaran][Ciaran] put up a [draft PR](https://github.com/puppetlabs/cisco_ios/pull/427) on some new Cisco/Bolt functionality. All comments welcome!
```

could add the "other work" section linking to https://tickets.puppetlabs.com/secure/RapidBoard.jspa?rapidView=1176&quickFilter=8745 to the template